badges/badges.css

```diff
@@ -0,0 +1,43 @@
+.disp-badges {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 1rem;
+}
+
+/* 100px = 6.25rem */
+.disp-badge-item {
```

Comment:
Try to use classnames with two words as mentioned by Aneena yesterday.

Reply:
Okay @TejasShekar . I'll look into it!